apps/atqr/web/src/services/interfaces.ts

```diff
@@ -1,65 +1,50 @@
-export enum SupervisorEnum {
-    'notInvited',
-    'invited',
-    'accepted',
-    'askedIfTheGoalIsAccomplished',
-    'repliedIfTheGoalWasAccomplished',
-  }
-
-  export enum PaymentMethodEnum {
-    'creditCard' = 'creditCard',
-    'debitCard' = 'debitCard',
-  }
-
-  export enum ChallengeStatus {
-    Ongoing = 'Ongoing',
-    Completed = 'Completed',
-    Failed = 'Failed',
-    Overdue = 'Overdue',
-    Created = 'Created'
-  }
+import { ChallengeStatus, JudgmentStatus } from '@atqr/domain';
 
-  export const updateStatus = (id: string, status: ChallengeStatus) => {
-    if(status == ChallengeStatus.Overdue){
-      return false
-    }else{
-      return status
-    }
+export enum PaymentMethodEnum {
```

Comment:
This enum is a part of another issue. You may delete it.

Reply:
Pesky enums, thought i've got them all! ahhahaahhahah. Done